code/modules/uplink/uplink_item.dm

```diff
@@ -1017,6 +1017,7 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 /datum/uplink_item/cyber_implants
 	category = "Cybernetic Implants"
 	surplus = 0
+	refundable = TRUE
```

Comment:
Undocumented change. Also, why?
I'm not sure, but I think this might let ops just use the empty box to refund all their TC making the implants free basically.